src/client/components/UserPage/CreateUrlModal/index.jsx

```diff
@@ -72,7 +72,7 @@ const CreateUrlModal = ({
   const incrementDecorator = (func) => async (...args) => {
     const proceed = await func(...args)
     if (proceed) {
-      setStep(step + 1)
```

Comment:
can consider just commenting the previous code out so in the future we know what we need to do to restore the old functionality